core/oa_hash_map.h

```diff
@@ -302,6 +337,9 @@ class OAHashMap {
 		return it;
 	}
 
+	OAHashMap(const OAHashMap &) = delete; // delete the copy constructor so we don't get unexpected copies and dangling pointers
+	OAHashMap &operator=(const OAHashMap &) = delete; // ditto for assignment operator
```

Comment:
Suggestion: start comments with a capital letter. Also, "ditto" is not used much around the code base, maybe replace it with "same"?

Reply:
Fair, I got that from another codebase :)

I'll fix the capitalization as well